### PR TITLE
Fix style and add links for Java GAE hello.

### PR DESCRIPTION
**Pull request by tswast**

### CODE REVIEW

File: java/gae-flexible-helloworld/README.md [hunk 1]
```diff
@@ -34,42 +34,44 @@ This app provides:
 
 1. Select **APIs & Auth > APIs**  
 
-1. Enable the **Cloud Bigtable API** and the **Cloud Bigtable Admin API**<br />
+1. Enable the [Cloud Bigtable API](https://console.cloud.google.com/apis/api/bigtable/overview) and the [Cloud Bigtable Admin API](https://console.cloud.google.com/apis/api/bigtabletableadmin/overview)<br />
   (You may need to search for the API.)
```
Comment (lesv) [inline java/gae-flexible-helloworld/README.md]: You could just have the link enable the API for you.

File: java/gae-flexible-helloworld/README.md [hunk 2]
```diff
@@ -34,42 +34,44 @@ This app provides:
...
 
-1. Select **Storage > Bigtable > Create Cluster**
+1. Select [Storage > Bigtable > Create Cluster](https://console.cloud.google.com/bigtable/zones/none/clusters/new).
```
Comment (lesv) [inline java/gae-flexible-helloworld/README.md]: Not sure how stable this URL will be.

File: java/gae-flexible-helloworld/README.md [hunk 3]
```diff
@@ -34,42 +34,44 @@ This app provides:
...
   Create a new Cluster -- You will need both the Zone and the Cluster ID
  
 1. Follow the [instructions to launch `HBase shell Quickstart`](https://cloud.google.com/bigtable/docs/quickstart)
 
 1. Create the table (tableName, Column Family)
 
- `create 'gae-hello', 'visits'`<br />
- `create 'from-json', 'cf1', 'cf2', 'cf3', 'cf4'`
- `exit`
+    ```
```
Comment (lesv) [inline java/gae-flexible-helloworld/README.md]: You might not need both spaces and back-apros, one or the other.

**Comment by lesv** [comment]
LGTM

File: java/gae-flexible-helloworld/README.md (hunk 1 above)
Comment (tswast) [inline java/gae-flexible-helloworld/README.md]: > You could just have the link enable the API for you.
Done.

File: java/gae-flexible-helloworld/README.md (hunk 2 above)
Comment (tswast) [inline java/gae-flexible-helloworld/README.md]: > Not sure how stable this URL will be.

Removed.

File: java/gae-flexible-helloworld/README.md (hunk 3 above)
Comment (tswast) [inline java/gae-flexible-helloworld/README.md]: > You might not need both spaces and back-apros, one or the other.

Done.

**Pull request merged**